.github/ISSUE_TEMPLATE/funding_application.yml

```diff
@@ -0,0 +1,221 @@
+---
+name: TI Funding Request Form
+description: TI Funding Request Form OpenSSF
+title: "[Technical Initiative Funding Request]: "
+labels:
+  - administration
+  - For Review
+  - Funding Request
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this Funding Request.
+  - type: markdown
+    attributes:
+      value: |
+        **Pre-conditions:**
+        - The TI must have an identified lifecycle phase inside the OpenSSF, such as sandbox, incubating, or graduated. 
+        - The TI requesters should know 
+          - Within general funding limits for the TIs at the respective lifecycle, that are to be determined. 
+          - Within approved categories of asks (e.g. no asking for a TI outing at a fancy restaurant but yes to cloud credits)
+  - type: markdown
+    attributes:
+      value: |
+        **Workflow:**
+        1. TI or TIs working together see a need for funding some work that doesn’t exist within an existing give/get that is pre-approved. For example cloud credits to do a PoC or case study.
+        2. TI(s) get consensus on the scope of how much money they are asking for and to be spent on what. For example $10k-$15k in cloud credits over a year to spend on building out a PoC to test out <some set of things>
+        3. TI(s) drafts that includes:
+          - Problem Statement in the form below
+          - Proposed work and other work considered
+          - Benefit to OSSF and ecosystem
+          - Approximate cost
```

Comment:
Is there an expectation that the TI would be responsible for vendor selection or would that responsibility fall to OpenSSF staff?

Reply:
I think TI should recommend what they'd prefer, but if funding is approved, they will work closely with OpenSSF Staff for final selection.

Reply:
The problem is that there may be a conflict of interest making such recommendations.

Reply:
If it is vendor-specific, it should be included in the funding request, (this only works on X). 

For non-vendor-specific, generic items like "cloud credits," staff can make recommendations based on what other projects use and the fee structures that the OpenSSF has access to in the larger LF ecosystem. I never see this as a forcing function (must use Y not X). The key is close communication with staff on needs. 

But this should probably be written into the process if it sounds appropriate.